File: Python/calculator.py
#!/bin/python3
# SPDX-License-Identifier: GPL-3.0-or-later
# (c) TechyMinati ( Aryan Sinha ) <sinha.aryan03@gmail.com>

def add(n, m):
    return n + m

def subtract(n, m):
    return n - m

def multiply(n, m):
    return n * m

def divide(n, m):
    return n / m


print("Select operation.")
print("1.Addition")
print("2.Subtract")
print("3.Multiply")
print("4.Divide")

while True:
    choice = input("Enter choice(1/2/3/4): ")

    if choice in ('1', '2', '3', '4'):
        num1 = float(input("Enter first number: "))
        num2 = float(input("Enter second number: "))

        if choice == '1':
            print(num1, "+", num2, "=", add(num1, num2))

        elif choice == '2':
            print(num1, "-", num2, "=", subtract(num1, num2))

        elif choice == '3':
            print(num1, "*", num2, "=", multiply(num1, num2))

        elif choice == '4':
            print(num1, "/", num2, "=", divide(num1, num2))
        
        next_calculation = input("Let's do next calculation? (yes/no): ")
        if next_calculation == "no" or "n":
          break
    
    else:
        print("Invalid Input Given, Please Retry :)")

File: C/basic/calculator.c
// !/bin/bash
// SPDX-License-Identifier: GPL-3.0-or-later
// (c) TechyMinati ( Aryan Sinha ) <sinha.aryan03@gmail.com>
// Simple Calculator in C

#include<stdio.h>
#include<math.h>
int main()
{
  float n1,n2;              // Lets Declare two variables here
  char operator;            // This is will store our operator
  printf("Enter an operator(+,-,/,*,^):");   // Take Operator Input from user 
  scanf("%c",&operator);    

  printf("Enter two numbers:");  // Take two numbers as input from User
  scanf("%f%f",&n1,&n2);    
  
  switch(operator)          // this code identifies which operator to use for operation, & displays output !
  {
      
    case'+' : printf("%.3f + %.3f = %.3f",n1,n2,(n1+n2)); 
              break;
    case'-' : printf("%.3f - %.3f = %.3f",n1,n2,(n1-n2));
              break;
    case'/' : printf("%.3f / %.3f = %.3f",n1,n2,(n1/n2));
              break;
    case'*' : printf("%.3f * %.3f = %.3f",n1,n2,(n1*n2));
              break;
    case'^' : printf("%.3f ^ %.3f = %.3f",n1,n2,pow(n1, n2));
              break;
    default: printf("Error! The given operator is wrong");

  }
  return 0;
}
